docker/utils.js

```diff
@@ -597,7 +597,7 @@ exports.printResultsToConsole = (
 		codeAuditorIssues.filter((x) => !!x.error).length > 0 ||
 		htmlErrors.length > 0
 	) {
-		consoleBox(`AUDIT FAIL`, 'red');
+		consoleBox(`AUDIT COMPLETE`, 'red');
```

Comment:
This is the required output (from the issue)

SSW Code Auditor - Scan Complete
Broken links: 7/310 - FAIL
Performance: 81/100 - FAIL
Accessibility: 100/100 - PASS